examples/c/minimal_legacy.bpf.c

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-2.0 OR BSD-3-Clause
+/* Copyright (c) 2020 Facebook */
```

Comment:
outdated and wrong Copyright?

Reply:
What do I change this to?

Reply:
You can either drop this line altogether or use current year and your name? Not a lawyer, but that's what I've seen done in practice :)